modules/setting/setting.go

```diff
@@ -915,6 +918,10 @@ func NewContext() {
 		}
 	}
 
+	if SSH.BuiltinSSHServerUser == "" {
+		SSH.BuiltinSSHServerUser = RunUser
```

Comment:
Couldn't we do simply SSH.BuiltinSSHServerUser = sec.Key("BUILTIN_SSH_USER").MustString(RunUser) here ?

Reply:
@sapk That's my question :D

Reply:
@Morlinest sorry I haven't look at others reviews. just quickly look at it ^^

Reply:
So my code is fine?

Reply:
What we suggest is that you move (and add the RunUser as default) L724 `SSH.BuiltinSSHServerUser = sec.Key("BUILTIN_SSH_USER").MustString(RunUser)` in place of L921-924

Reply:
@sapk It cannot happen because sec is redefined in between those spots. That's why I have the two lines.

Reply:
@sapk It cannot happen because sec is redefined in between those spots. That's why I have the two lines.

Reply:
`Cfg.Section("server").Key("BUILTIN_SSH_SERVER_USER").MustString("")`?

Reply:
@Morlinest I didn't know about that one. Thanks. I'll try it.